internal/cmd/db_shell.go

```diff
@@ -67,117 +46,75 @@ var shellCmd = &cobra.Command{
 			return err
 		}
 
-		token, err := tokenFromDb(db, client)
+		token, err := tokenFromDb(&db, client)
 		if err != nil {
 			return err
 		}
 
 		dbUrl := nameOrUrl
-		if db != nil {
```

Comment:
Why is it now safe not to check whether db is not nil?

Reply:
I forced an error inside `databaseFromNameOrURL` if we can't find the database